internal/clients/ec2/aws_supported_types.go

```diff
@@ -0,0 +1,45 @@
+package ec2
+
+import (
+	"context"
+
+	"github.com/RHEnVision/provisioning-backend/internal/clients"
+	"github.com/RHEnVision/provisioning-backend/internal/payloads"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+)
+
+func FilterAWSInstanceTypes(_ context.Context, it *[]clients.InstanceType) {
+	for _, i := range *it {
+		if IsSupported(i) {
+			// If the Instance type is present in the yml it is supported
+			i.Supported = true
+		}
+	}
+}
+
+func IsSupported(i clients.InstanceType) bool {
+	_, ok := clients.AwsSupported[i.Name]
+	return ok
+}
```

Comment:
Please move this function to the clients package. Also unexport `AwsSupported` variable, it's definitely something that callers should not see.